terraform/modules/bap_alerting_gcp/opensearch.tf

```diff
@@ -0,0 +1,62 @@
+# Basic OpenSearch metrics and alerts
+
+resource "google_logging_metric" "opensearch_server_count" {
+  name        = "opensearch.server.count"
+  description = "Number of OpenSearch server"
+
+  count         = var.alerts ? 1 : 0
+
+  filter      = "logName =~ \"opensearch-logs$\" AND jsonPayload.level: (\"INFO\" OR \"WARN\")"
+}
+
+# resource "google_logging_metric" "opensearch_server_stopped" {
+#   name        = "opensearch.server.stopped"
+#   description = "When OpenSearch server receive the stopped message"
+
+#   count         = var.alerts ? 1 : 0
+
+#   filter      = "logName : \"opensearch-logs$\" AND jsonPayload.message: \"stopped\" AND jsonPayload.level: (\"INFO\")"
+# }
+
+# resource "google_logging_metric" "opensearch_server_started" {
+#   name        = "opensearch.server.started"
+#   description = "When OpenSearch server receive the started message"
+
+#   count         = var.alerts ? 1 : 0
+
+#   filter      = "logName : \"opensearch-logs$\" AND jsonPayload.message: \"started\" AND jsonPayload.level: (\"INFO\")"
+# }
```

Comment:
Why are these commented?

Reply:
These could be good metrics but we don't have alerts for these. I remove them